examples/python/IRD_stream.py

```diff
@@ -61,8 +61,9 @@
 ### FLAT (for blaze function) ###
 flat_mmf.trace = trace_mmf
 flat_mmf.clean_pattern(extin='', extout='_cp', hotpix_mask=hotpix_mask)
-flat_mmf.flatten(imcomb=True)
-flat_mmf.dispcor(imcomb=True)
+flat_mmf.imcomb = True # median combine
```

Comment:
What does `imcomb=True` mean?

Reply:
It means to apply flatten() or dispcor() to a median combined image of multiple files. Originally, these functions could only be used for a single file, so I changed this way to solve that. (but there may be a better way to implement it...)